helm_deploy/values-dev.yaml

```diff
@@ -35,6 +35,7 @@ generic-service:
     PATHFINDER_UI_URL: "https://dev.pathfinder.service.justice.gov.uk"
     USE_OF_FORCE_UI_URL: "https://dev.use-of-force.service.justice.gov.uk"
     WELCOME_PEOPLE_INTO_PRISON_UI_URL: "https://welcome-dev.prison.service.justice.gov.uk"
+    LEARNING_AND_WORK_PROGRESS_URL: "https://learning-and-work-progress-dev.hmpps.service.justice.gov.uk"
```

Comment:
Minor; might be worth naming this `LEARNING_AND_WORK_PROGRESS_UI_URL` so it is consistent with the other envs vars in this section for UIs